submissions/egwoloOkorodudu/HTML/Contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>contact page</title>
+    </head>
+    <header>
+        <h1>Dudu's Grill</h1>
+    </header>
+    <body>
+    
+        <nav>
+        <a href="../HTML/Product.html">Product Page</a>
+
+        </nav>  
+
+        <h3> Contact Us</h4> 
+            Please fill this form<br>
+            <label for="full name">Full Name</label><br>
+            <input type="text" name="full name" id="full name"/><br><br/>
+            <label for="email">Email</label><br>
+            <input type="email" name="email" id="email"/><br><br/>
+            <label for="subject">Subject</label><br>
+            <input type="text" name="subject" id="subject"/><br><br/>
+
+            <label for="message">Message</label><br>
+            <textarea name="message" id="message" cols="40" rows="10"></textarea><br><br/> 
+            <button>Submit</button><br><br/>
```

Comment:
Remove all the spaces from line 29 to 43

Reply:
I actually link my pages together.
lost my my folder from the submission folder.